src/rich-text/commands/index.ts

```diff
@@ -104,10 +105,38 @@ export function insertHorizontalRuleCommand(
         return false;
     }
 
-    dispatch &&
+    if (!dispatch) return true;
```

Comment:
nit: formatting. I'm surprised I don't have eslint or prettier set up to detect this one...

```suggestion
    if (!dispatch) {
        return true;
    }
```